src/services/export.service.js

```diff
@@ -11,7 +11,7 @@ const bulkExport = async (request, reply) => {
   if (validateExportParams(request, reply)) {
     request.log.info('Base >>> $export');
     const clientEntry = await addPendingBulkExportRequest();
-    exportToNDJson(clientEntry, request);
+    await exportToNDJson(clientEntry, request, reply);
```

Comment:
we don't actually want to `await` this. We intentionally left it out so it would kick off in the background and not block returning from the initial kickoff request.

```suggestion
    exportToNDJson(clientEntry, request, reply);
```